### ISSUE TITLE
mzn2fzn crashes when dealing with float vars

### ISSUE THREAD

**Issue by claudix** [MiniZinc/libminizinc#117]
The simple program below makes `mzn2fzn` crash: Windows application crash message appears and IDE shows the message **"Unknown error while executing the MiniZinc interpreter `mzn2fzn`: error code 1"**.

```
array[1..2] of var {5.0, 6.0}: v;  
constraint v[1] + v[2] > 0;
solve satisfy;
```

**Comment by a1880** [comment]
I have run this snippet with `mzn2fzn` version `2.0.14` under Windows 7, 64bit

`mzn2fzn --verbose xxx.mzn`

The output is:

```
Parsing 'xxx.mzn'
processing file 'xxx.mzn'
processing file 'C:\AK\Programme\MiniZinc_IDE_2.0.14/share/minizinc/std/stdlib.mzn'
processing file 'C:\AK\Programme\MiniZinc_IDE_2.0.14/share/minizinc/std/builtins.mzn'
processing file 'C:\AK\Programme\MiniZinc_IDE_2.0.14/share/minizinc/std/redefinitions-2.0.2.mzn'
processing file 'C:\AK\Programme\MiniZinc_IDE_2.0.14/share/minizinc/std/redefinitions-2.0.mzn'
processing file 'C:\AK\Programme\MiniZinc_IDE_2.0.14/share/minizinc/std/redefinitions.mzn'
processing file 'C:\AK\Programme\MiniZinc_IDE_2.0.14/share/minizinc/std/flatzinc_builtins.mzn'
Done parsing (30 ms)
Typechecking ... done (6 ms)
Flattening ...
```

Then, an exception occurs and the program comes to an end.

In Visual Studio 2015, the error message ist:

`Unhandled exception at 0x01445033 in mzn2fzn.exe: 0xC0000005: Access violation reading location 0x0000001C.`

**Comment by guidotack** [comment]
This has been fixed in a478cf5ea3857f59db41b91abf8d821c7345ebbc.
